### PR TITLE
Update deps to latest patch versions

### PR DESCRIPTION
**Pull request by dgn**

### CODE REVIEW

**Comment by howardjohn** [comment]
I looked into the failure, pretty sure its a k8s bug

File: go.mod [hunk 1]
```diff
@@ -85,51 +75,39 @@ require (
...
-	github.com/hashicorp/serf v0.8.1 // indirect
-	github.com/hashicorp/vault v0.10.0
+	github.com/hashicorp/golang-lru v0.5.4 // indirect
+	github.com/hashicorp/serf v0.8.5 // indirect
+	github.com/hashicorp/vault/api v1.0.3
 	github.com/howeyc/fsnotify v0.9.0
-	github.com/huandu/xstrings v1.2.0 // indirect
-	github.com/imdario/mergo v0.3.7 // indirect
+	github.com/huandu/xstrings v1.2.1 // indirect
+	github.com/imdario/mergo v0.3.8 // indirect
```
Comment (howardjohn) [inline go.mod]: revert this and things will work

Comment (dgn) [reply go.mod]: Thank you John! That does seem to do the trick. Strangely, the README.md of that repo lists 0.3.7 as the latest stable release, but they still have a 0.3.8 tag (which lead `go get -u=patch` to update it)

Comment (howardjohn) [reply go.mod]: Yeah I was confused by that as well. I checked k8s is using an older version so thats why its not noticed most likely

**Pull request merged**